### DISCUSSION TITLE
Managing blog and content

### DISCUSSION THREAD

**Discussion by Twambilile** [dlarroder/dalelarroder#159] [Q&A] [answered]
Hello

I wanted to ask how can i manage the blog and post different blogs. I currently have just been able to fork the repo but I am unable to do anything else apart from changing static info information. What steps must I take in order to better utilize the blog and everything else? Please help

**Comment by dlarroder** [comment, marked as answer]
Hey @Twambilile, you can manage the blog data in the `data/blog` directory, if you want to add a new blog entry, you can just create a new mdx file and add the required fields like:

![Screen Shot 2023-02-11 at 12 22 05 AM](https://user-images.githubusercontent.com/52998821/218142691-f526d299-d4a3-47c6-88ca-e14abb06f670.png)

After adding a new mdx file, you'll automatically have a new page in the `/blog` page with it,  in this case it's https://www.dalelarroder.com/blog/why-does-react-need-keys

When deleting, it's the same, if you delete the `why-does-react-need-keys.mdx` file, it will automatically delete it from the site.

Hope this helps!

**Reply by Twambilile** [reply]
Oh, thank you very much! This was exactly what I needed help with from you, appreciate it!
